ui/src/features/BottomPanel/components/Console/index.tsx

```diff
@@ -1,6 +1,6 @@
 const LogConsole: React.FC = () => {
   return (
-    <div className="bg-zinc-900 text-yellow-600 text-xs h-[210px] w-[100%] overflow-scroll rounded-md p-1">
+    <div className="bg-zinc-900 text-yellow-600 text-xs h-[400px] w-[100%] overflow-scroll rounded-md p-1">
```

Comment:
I think h-full is better here so that it takes up its parent's height (the BottomPanel, where you are defining the height)

Reply:
I was also looking into this. I want to work on this in the Maximize view PR, is that okay?